README.md

```diff
@@ -27,6 +27,6 @@ Multi-User) illustrates the different forms of engagement Maroon supports or pla
 
 # Simulations and Experiments
 
-Maroon contains several physics experiments, but will be extended with computer science simulations and experiences from other subjects in the future. The physics experiments cover Van de Graaff Generator, Falling Coil, Faraday’s Law, Capacitor and Huygen's Principle.
+Maroon contains several physics, chemistry and computer science experiments, but will be extended with experiences from other STEM fields in the future. The physics experiments cover Coulomb's Law, Falling Coil, Faraday's Law, Huygen's Principle, Point Wave, Pendulum, Van de Graaff Balloon, Van de Graaff Generator, Optics, 3D Motion Simulation and Cathode Ray Tube. The chemistry experiments are represented by Titration and Catalyst. And Computer Science experiments include Sorting, State Machine, Path Finding and Minimum Spanning Tree.
```

Comment:
"physics, chemistry, and computer science" (missing comma after chemistry)

Physics also includes e.g. Falling Ball Viscosimeter; (And also missing comma before the "and"), i.e. should be "3D Motion Simulation, Cathode Ray Tube, and Falling Ball Viscosimeter."
Computer science also includes e.g Perlin Noise; and also missing comma before "and"

Reply:
Oh, fair! I actually thought I took all the experiments that were listed. Just updated it!